build/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.7.1
+
+- Add a top-level `log` getter which is scoped to running builds and can be used
+  anywhere within a build rather than pass around a logger. This replaces the
```

Comment:
In practice what would be the scope of the top level getter log? It is within the build steps right? Otherwise the name seems too generic.